main/ertp/guide/README.md

```diff
@@ -1,41 +1,334 @@
+  The source of truth of how many digital assets each `purse` and `payment` holds. An `issuer`
+  is used to validate `payments` received from untrusted parties. Specifically, it validates
+  `payments` of the `brand` the `issuer` is associated with. Has a one-to-one relationship
+  with both a `brand` and a `mint`. 
+- **[AmountMath](./AmountMath.md)** ([glossary](https://agoric.com/documentation/glossary/#amountmath)):
+  Methods to do math operations on `amounts`. Each `brand` can be associated with many `amountMaths`,
+  but each `amountMath` is permanently associated with only one `brand`.
 
-## Guide
+![ERTP object relationships](./assets/relationships1.svg) 
 
-Let's get started by creating some different kinds of digital assets:
-- [Creating a fungible asset with ERTP](./create-fungible-erights.md)
-- [Creating non-fungible tokens with ERTP](./create-non-fungible-tokens.md)
+Let's look at an example. Suppose there is the "Quatloos" `brand`. That means there is also:
+- A "Quatloos `mint`" that is the only ERTP `mint` that can ever create new Quatloos assets.
+- A "Quatloos `issuer`" that is the only `issuer` that can create a new `purse` to contain Quatloos and 
+  operate on a `payment` containing Quatloos.
+- "Quatloos `amountMaths`" whose operations only work on an `amount` whose `brand` is Quatloos.
+
+![ERTP object relationships 2](./assets/relationships2.svg) 
+
+We've already mentioned our final two components:
+- **[Purse](./PursesAndPayments.md)** ([glossary](https://agoric.com/documentation/glossary/#purse)): An
+  object for holding digital assets of a specific `brand`.
+- **[Payment](./PursesAndPayments.md)** ([glossary](https://agoric.com/documentation/glossary/#payment)):
+  An object for transferring digital assets of a specific `brand` to another party.
+  
+Similar to other component instances, a `purse` and a `payment` only work with one
+`brand`. So a `purse` or `payment` that holds Quatloos cannot hold an asset of `brand` Moola or vice versa. 
+You cannot change the `brand` a `purse` or `payment` was originally associated with. Once you create a
+"Quatloos `purse`" or "Quatloos `payment`", they can never hold anything other than Quatloos.
+
+However, these are not one-to-one relationships. There can be thousands or more
+`purses` or `payments` that hold Quatloos or any other `brand`.
+
+## Method Naming Structure
+
+ERTP methods use a template for their names. Knowing what a particular method name prefix represents
+can help you when reading code.
+
+- `make<Foo>()`: Creates a new Foo object and returns only that object.
+- `make<Foo>Kit()`: Creates a new Foo object as well as other things. It returns some combination of useful things, usually including the new
+  Foo object. But not always; sometimes Foo is conceptual, and, for example, instead of a single object, two facets are returned.
+- `create<Foo>()`: Creates a new Foo, but doesn't return it. 
```

Comment:
This was an overall naming schema Chris suggested adding. I don't think we use all of them, but it might be a good idea to keep them as a guideline for developer-defined method naming?

Reply:
I think it makes sense to include the method patterns that we use since that's evidence of buy-in, but I'm not sure that we have agreement on this method pattern, since it's not in use. Maybe we do?